Updated the about page text (#522)
mitodl/ocw-hugo-themes: www/content/milestones/2016-2020.md

image: /images/about/timeline-image-2.png
headless: true
milestones:

>   - OCW becomes the most-subscribed .edu channel on YouTube.

Reviewer: add link to https://www.youtube.com/c/mitocw

```suggestion
  - OCW becomes the most-subscribed .edu channel on [YouTube](https://www.youtube.com/c/mitocw).
```

>   - The 200th Instructor Insights video is published through the OCW Educator program.
>   - Interactive, searchable transcripts are added to videos.

Reviewer: remove these two lines
```suggestion
```

>   - New MIT Open Learning Library launches in 2019, offering new, self-paced learning options complementing OCW and MITx.

Reviewer: link to https://openlearning.mit.edu/courses-programs/open-learning-library
```suggestion
  - New [MIT Open Learning Library](https://openlearning.mit.edu/courses-programs/open-learning-library) launches in 2019, offering new, self-paced learning options complementing OCW and MITx.
```

>   - Chalk Radio podcast launches in February 2020.

Reviewer: add link
```suggestion
  - [Chalk Radio podcast](https://chalk-radio.simplecast.com/) launches in February 2020.
```

>   - OCW website traffic nearly doubles as the COVID-19 pandemic disrupts life around the world.

Reviewer: ```suggestion
  - OCW supports learners around the world as the COVID-19 pandemic disrupts education.
```
